tools/cert_create/Makefile

```diff
@@ -10,6 +10,7 @@ V		:= 0
 DEBUG		:= 0
 BINARY		:= ${PROJECT}${BIN_EXT}
 OPENSSL_DIR	:= /usr
+USE_TBBR_DEFS	?= 1
```

Comment:
When Make is invoked from the top Makefile, `USE_TBBR_DEFS=${USE_TBBR_DEFS}` is passed correctly down to tools/cert_create/Makefile, so `USE_TBBR_DEFS ?= 1` is not effective.

`USE_TBBR_DEFS ?= 1` was added here in order to avoid build failure when cert_create is built directly in tools/cert_create directory.

You said as follows in pull request #837:
> (that would prevent passing a command line CFLAGS to the tools Makefile when invoked directly (although admittedly this isn't properly supported anyway)

Seemed you liked to cater to such a use-case, so I fixed the build error just in case.

In upstream, building in tools/cert_create never succeeds:

    $ cd tools/cert_create; make
    ../../make_helpers/plat_helpers.mk:56: *** "Error: Invalid platform. The following platforms are available: fvp|juno|mt6795|mt8173|qemu|rk3368|rk3399|tegra|zynqmp".  Stop.

If you prefer to keeping backward compatibility including the build failure, I can change it to `USE_TBBR_DEFS ?= 0` to get the build fail again.